### PR TITLE
convertFeetToMeters.js and Test class renamed to correct class name

### LINKED ISSUE #374: Fix convertFeetToMeters method
There is currently a method called `convertFeetToMeters` that lives in `/lib/convertFeetToMeters.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test <methodName>` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by Aonepathan**
Closes #374  

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing

Will be adding Acceleration.js and test class soon!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,5 +31,8 @@
   },
   "pre-commit": [
     "lint"
-  ]
+  ],
```
Comment (ktilcu) [inline package.json]: lets remove these changes

File: yarn.lock [hunk 2]
```diff
@@ -1460,6 +1460,10 @@ functional-red-black-tree@^1.0.1:
   version "1.0.1"
   resolved "https://registry.yarnpkg.com/functional-red-black-tree/-/functional-red-black-tree-1.0.1.tgz#1b0ab3bd553b2a0d6399d29c0e3ea0b252078327"
 
+ganon@^2.0.19:
```
Comment (ktilcu) [inline yarn.lock]: These too

**Comment by Aonepathan** [comment]
Hello @ktilcu I've made the necessary changes you mentioned and also added a averageAcceleration.js skeleteon, please let me know if further changes are needed! Question, the Travis CI build fails because the test class is expecting results from the skeleton class, what should I do to let the test class pass the Travis CI Build?

**Comment by ktilcu** [comment]
@Aonepathan sorry. We had merged a change to update our buiild process to try to run tests on travis. I have reverted that commit. If you rebase and push then you should be good to go!

**Comment by Aonepathan** [comment]
@ktilcu Sounds good, I've updated the PR, should be good now? Let me know if any changes need to be made!

**Pull request merged**